### PR TITLE
Seed cluster support for dogfooding

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: handler/user.go [hunk 1]
```diff
@@ -7,22 +7,54 @@ import (
 	"github.com/dgrijalva/jwt-go"
 	"github.com/golang/glog"
 	"github.com/gorilla/context"
+"github.com/kubermatic/api/provider"
```
Comment (s-urbaniak) [inline handler/user.go]: goimports

File: provider/types.go [hunk 2]
```diff
@@ -16,6 +16,11 @@ const (
 	LinodeCloudProvider       = "linode"
 )
 
+type User struct {
+	Name string
+	Roles map[string]*struct{}
```
Comment (s-urbaniak) [inline provider/types.go]: why `*struct{}`?

**Comment by s-urbaniak** [comment]
nits, i don't like the `*struct{}`

**Comment by sttts** [comment]
All addressed.

**Pull request merged**